.devcontainer/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Use a base image that includes both Go and Node.js, such as the official Go image
+FROM golang:latest
+
+
+ENV GOPATH=/go
+ENV PATH=$PATH:$GOPATH/bin
+ENV AVALANCHEGO_EXEC_PATH=$GOPATH/src/github.com/ava-labs/avalanchego/build/avalanchego
+ENV AVALANCHEGO_PLUGIN_PATH=$GOPATH/src/github.com/ava-labs/avalanchego/build/plugins
+
+# Install Node.js and npm using the official Node.js image
+RUN curl -fsSL https://deb.nodesource.com/setup_20.x | bash - && apt-get install -y nodejs
+
+# Clone the avalanchego repository
+RUN git clone -b v1.10.5 https://github.com/ava-labs/avalanchego.git $GOPATH/src/github.com/ava-labs/avalanchego
```

Comment:
can we use an ARG for the avalanchego version here?

Reply:
I think if we can use the same Dockerfile as in Subnet-EVM [here](https://github.com/ava-labs/subnet-evm/blob/master/Dockerfile), we should just use that.

Reply:
also you can take a look at build_image script [here](https://github.com/ava-labs/subnet-evm/blob/master/scripts/build_image.sh) to see how we build docker images.

Reply:
> I think if we can use the same Dockerfile as in Subnet-EVM [here](https://github.com/ava-labs/subnet-evm/blob/master/Dockerfile), we should just use that.

This dockerfile does not include Avalanche Network Runner and does not set the environment variables

Reply:
We should be using the avalanchego version from `scripts/version.sh`. Otherwise it could be very easy to miss changing this line in Dockerfile while updating the avalanchego.